Makefile.toml

```diff
@@ -532,3 +540,22 @@ kill $shard_manager_pid || true
 kill $router_pid || true
 kill $redis_pid || true
 '''
+
+[tasks.generate-configs]
```

Comment:
As this is a manual step that we will forget to run, we could add a check to CI similar to how we check that the generated OpenAPI yaml is up-to-date.

Reply:
https://github.com/golemcloud/golem/pull/620/commits/18093028d40a4bdf4e740ee1877e09a4e856eb99